modules/commons/pom.xml

```
@@ -42,6 +42,7 @@
             <!-- Initial content to be loaded on bundle installation -->
             <Sling-Initial-Content>
               SLING-INF/content/libs/lfs/Resource/;path:=/libs/lfs/Resource/;overwrite:=true,
+              SLING-INF/content/libs/slings/errorhandler/;path:=/libs/slings/errorhandler/;overwrite:=true,
```

```suggestion
              SLING-INF/content/libs/slings/servlets/errorhandler/;path:=/libs/slings/servlets/errorhandler/;overwrite:=true,
```

Also move the jsp file.